packages/datadog-plugin-sharedb/test/index.spec.js

```diff
@@ -0,0 +1,194 @@
+'use strict'
```

Comment:
Errors and context propagation should be tested in addition to the current tests.

Reply:
Will try to make sharedb blow up to test error handling, and also add tests for context propagation.